src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5152,7 +5156,7 @@ internal sealed class ManifestBuilder
         }
 
         /// <summary>
-        /// <term>Will NOT build a manifest!</term> If the intention is to build a manifest don�t use this constructor.
+        /// <term>Will NOT build a manifest!</term> If the intention is to build a manifest don�t use this constructor.
```

Comment:
Revert?

Reply:
Hmm I see now the previous one was also a special character.
```suggestion
        /// <term>Will NOT build a manifest!</term> If the intention is to build a manifest don't use this constructor.
```

Reply:
Yeah not in the scope anyway :)